.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: "CCF App Template CI"
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build-app:
+    runs-on: ubuntu-latest
```

Comment:
I suggest we set 20.04 explicitly, since that's what we/Open Enclave support officially